src/neg_tests/domain_lin_tests.ml

```diff
@@ -5,7 +5,7 @@ open Lin_tests_common
 ;;
 QCheck_base_runner.run_tests_main
   (let count = 15000 in
-   [RT_int.neg_lin_test    `Domain ~count ~name:"Lin ref int test with Domain";
-    RT_int64.neg_lin_test  `Domain ~count ~name:"Lin ref int64 test with Domain";
-    CLT_int.neg_lin_test   `Domain ~count ~name:"Lin CList int test with Domain";
-    CLT_int64.neg_lin_test `Domain ~count ~name:"Lin CList int64 test with Domain"])
+   [RT_int_domain.neg_lin_test    ~count ~name:"Lin_api ref int test with Domain";
+    RT_int64_domain.neg_lin_test  ~count ~name:"Lin_api ref int64 test with Domain";
+    CLT_int_domain.neg_lin_test   ~count ~name:"Lin_api CList int test with Domain";
+    CLT_int64_domain.neg_lin_test ~count ~name:"Lin_api CList int64 test with Domain"])
```

Comment:
These are not `Lin_api` tests so I think this test renaming is a mistake.

In the "old terminology":
- `domain_lin_tests.ml` depend on `lin_tests_common.ml` and are `Lin` tests
- `domain_lin_tests_dsl.ml` depend on `lin_tests_common_dsl.ml` and are `Lin_api` tests

I think this is caused by the confusing test file names, hence I renamed them in #197.
With the new file names
- the former would share a common `lin_tests_` prefix
- the latter would share a common `lin_tests_dsl_` prefix

Reply:
Well spotted, sorry about that. I’m afraid it’s probably more due to an ill-advised copy-paste when cherry-picking that commit on `main`.